autogpt/plugins.py

```diff
@@ -230,12 +230,21 @@ def scan_plugins(cfg: Config, debug: bool = False) -> List[AutoGPTPluginTemplate
                         continue
                     a_module = getattr(zipped_module, key)
                     a_keys = dir(a_module)
+
+                    is_plugin = False
+                    if "_abc_impl" in a_keys:
+                        for base in a_module.__bases__:
```

Comment:
Add logging for when we don't load a plugin bc that template isn't there

Reply:
thats nonsense, that would basically add the claim that a module wasn't loaded for any module thats not a plugin, which will be extremly confusing to the user. The whole point of this PR is to make a difference between any random python  module and a plugin

Reply:
I understand what you're saying but this will probably break a whole lot of existing plugins so giving a debug message would probably be appreciated by some devs

Reply:
all plugins listed (first and 3rd party) work, none of them would break, where do you get "a whole lot of existing plugins" from?